bin/release_merged_branch

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env ruby
+
+require "bundler/inline"
+gemfile do
+  source "https://rubygems.org"
+  gem "multi_repo", require: "multi_repo/cli"
+end
+
+require "fileutils"
+
+opts = Optimist.options do
+  opt :branch,        "The branch to merge into.",              :type => :string, :required => true
+  opt :source_branch, "The source branch from which to merge.", :default => "master"
+
+  opt :review_only,   "Skip the merge and re-review the branch details", :default => false
```

Comment:
Should this be `dry_run` instead?

Reply:
Not really, it's slightly different, but maybe?

The way the code works is that it always makes changes to the local repositories, then it goes to review, and then it shows a list of command for manually pushing.

So, dry-run doesn't really exist because it always makes local changes. review-only is saying "skip the first part and go directly to the review again"

It doesn't make sense to me to reuse the dry-run flag for this case, because the changes did already happen (and can actually be pushed).